doc/userguide/rules/ike-keywords.rst

```diff
@@ -0,0 +1,159 @@
+IKE Keywords
+==============
+
+The keywords
+
+* ``ike.spi_initiator``
+* ``ike.spi_responder``
+* ``ike.chosen_sa_attribute``
+* ``ike.exchtype``
+* ``ikev1.vendor``
+* ``ikev1.server_key_exchange_payload``
+* ``ikev1.client_key_exchange_payload``
```

Comment:
Could there be only `ikev1.key_exchange_payload` ? cf https://github.com/OISF/suricata/pull/5097#discussion_r477249870